kong/init.lua

```diff
@@ -508,10 +617,16 @@ function Kong.rewrite()
 
   runloop.rewrite.before(ctx)
 
+  local ok = plugins_map_wrapper()
```

Comment:
Any reason why we are not invoking this from `ssl_certificate` as well? My understanding is that `rewrite` catches non-SSL connections (that's great), but when a new connection is opened just after a new plugin was configured, then are we failing to execute it during the SSL handshake?

Reply:
Gooood catch. I didn’t think about this, largely because we have no plugins that uses ssl phase. 

Tangentially- to my understanding the ssl phase historically was provided for plugins so we could use dynamic ssl before ssl_by_lua was a Thing. Given that we have certs and snis as first class objects, should this phase handler for plugins go away?